react/src/pages/RoutingListPage.tsx

```diff
@@ -125,13 +126,16 @@ const RoutingListPage: React.FC<RoutingListPageProps> = () => {
             resource_group
             resource_slots
             resource_opts
+            created_user_email @since(version: "23.09.8")
+            session_owner_email @since(version: "23.09.8")
```

Comment:
We can remove this because these fields aren't used in RoutingListPage. It is used EndpointOwenInfo.
```suggestion

```